reframe/frontend/cli.py

```diff
@@ -475,6 +477,8 @@ def main():
             mode_args = site_config.get(f'modes/@{options.mode}/options')
 
             # Parse the mode's options and reparse the command-line
+            mode_args = list(itertools.chain(*[shlex.split(m)
+                                               for m in mode_args]))
```

Comment:
```suggestion
            mode_args = list(itertools.chain(*(shlex.split(m)
                                               for m in mode_args)))
```
I am not sure whether it makes a different, @vkarak is this better than the temporary list?

Reply:
```suggestion
            mode_args = list(itertools.chain.from_iterable(shlex.split(m) for m in mode_args))
```
Or this? 🤔

Reply:
The second one is the most pythonic.